src/main/java/com/nitnelave/CreeperHeal/block/CreeperChest.java

```diff
@@ -125,8 +125,7 @@ private boolean hasNeighbor()
     public void update()
     {
         super.update();
-        getBlock().setType(blockState.getType());
-        getBlock().setData(blockState.getRawData());
+        blockState.update(true, false); // This makes the original block state update and doesn't call physics.
```

Comment:
How is that fundamentally different from the original code? How will that prevent the error from happening? setType should already make the block a chest, but it's not the case here.